lib/elements/number.js

```diff
@@ -55,7 +56,21 @@ class NumberPrompt extends Prompt {
       this.placeholder = false;
       this.rendered = this.transform.render(`${round(v, this.round)}`);
       this._value = round(v, this.round);
+      
+      if (this.resetDelay > 0) {
```

Comment:
For params having to do with time `0` defaults are generally safer. This way it is clear to the user that a `0` delay is impossible. 
Unlike the case with the `indexOf()` method where a `0` result is possible and `-1` result means an element is not found.

Reply:
Ok, I think that makes sense to me, but to clarify -- the idea is that if the user sets `resetDelay = 0` they expect there will be _no_ delay timer and the number will _not_ automatically reset at any time while they are typing? And in this case, it would be clearer to document the default value as being `0` instead of a negative number like `-1`, correct?

Reply:
updated the default params and docs to `0` per this conversation